mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -758,6 +759,8 @@ void deleteMailboxByIdShouldFireMailboxDeletionEvent() throws Exception {
 
         @Test
         void createMailboxShouldFireMailboxAddedEvent() throws Exception {
+            groupListener.reset(); //needed as setUp did lead to events being dispatched
```

Comment:
is it because we register the EventCollector earlier? What about calling `.reset` in `setUp` then?